docker-compose.yml

```diff
@@ -224,6 +224,37 @@ services:
       - host.docker.internal:${DOCKER_HOST:-host-gateway}
     restart: always
 
+  collaboration:
```

Comment:
we also need separate service configs for onlyoffice and collabora. And we have to think how to do it. via `.env` or `profiles`